internal/pkg/cli/pipeline_init.go

```diff
@@ -397,12 +413,12 @@ func (opts *InitPipelineOpts) getGitHubAccessToken() error {
 	return nil
 }
 
-// TODO: nice to have a remote GitHub repo's branch list to offer a select menu.
 func (opts *InitPipelineOpts) getGitHubBranch() error {
-	branch, err := opts.prompt.Get(pipelineEnterGitHubBranchPrompt,
-		"Name of the branch that you wish to use in your GitHub repository. By default the branch name is \"master\".",
-		nil, prompt.WithDefaultInput("master"))
-
+	branches, err := opts.listGitHubBranches()
```

Comment:
Should we make the branch flag to be optional, and default to "master" as I assume that'll be the most common use-case. This way we can remove the logic to query for branch names.

Reply:
Yeah that's a good idea